cinema/permissions.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework.exceptions import NotFound, MethodNotAllowed
+from rest_framework.permissions import BasePermission, SAFE_METHODS
+
+
+class IsAdminOrIfAuthenticatedReadOnly(BasePermission):
+    """The request is authenticated as an admin - read/write,
+    if as a user - read only request"""
+
+    def has_permission(self, request, view):
+
+        return bool(
+            request.method in SAFE_METHODS
+            and request.user.is_authenticated
+        ) or request.user.is_staff
```

Comment:
"in" operator returns boolean, is_authenticated as well, so what is the purpose to convert it to bool?